Meta/CMake/vcpkg/overlay-ports/icu/darwin-rpath.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/source/config/mh-darwin b/source/config/mh-darwin
+index 7b15709..e2cdbdb 100644
+--- a/source/config/mh-darwin
++++ b/source/config/mh-darwin
+@@ -31,7 +31,8 @@ SHLIB.cc=	$(CXX) -dynamiclib -dynamic $(CXXFLAGS) $(LDFLAGS) $(LD_SOOPTIONS)
+ 
+ ## Compiler switches to embed a library name and version information
+ ifeq ($(ENABLE_RPATH),YES)
```

Comment:
In the overlay, I needed to include a revert of https://github.com/unicode-org/icu/commit/b542ae9d9123231d89ab0694e5809c9dfa45c8f7 as my system autoconf was too old to support that version on Ubuntu LTS.  I suspect we will need to still revert that.

Reply:
A revert for that is already included ([Meta/CMake/vcpkg/overlay-ports/icu/ladybird-remove-autoconf272.patch](https://github.com/LadybirdBrowser/ladybird/pull/3295/commits/d7141bba941951dbb5a361f7284fbb340aed78d5#diff-ad7f90ab06223acecfc134375894444ec1f68f1c82b9636b60c31e7e341df020)). I didn't hit it on my system, but CI did.

Reply:
Ah, sorry I missed it going through the patches